app/Actions/Docker/GetContainersStatus.php

```
@@ -418,81 +436,98 @@ public function handle(Server $server, ?Collection $containers = null, ?Collecti
-                    if ($excludeFromHc || $restartPolicy === 'no') {
-                        $excludedContainers->push($serviceName);
-                    }
-                }
-            } catch (\Exception $e) {
-                // If we can't parse, treat all containers as included
-            }
-        }
+        $excludedContainers = $this->getExcludedContainersFromDockerCompose($dockerComposeRaw);
 
         // Filter out excluded containers
         $relevantStatuses = $containerStatuses->filter(function ($status, $containerName) use ($excludedContainers) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$status'. (undefined)

(UnusedFormalParameter)